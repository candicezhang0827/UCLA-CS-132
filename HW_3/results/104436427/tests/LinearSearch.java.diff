1,4c1,15
< --- Error (ProgramFault) ---
< call to "LS.Print": incorrect number of arguments in function call (expecting: 1, found: 2)
<   Main() line 14
<   LS.Start(<Heap.0>+0 10) line 24
---
> 10
> 11
> 12
> 13
> 14
> 15
> 16
> 17
> 18
> 9999
> 0
> 1
> 1
> 0
> 55
