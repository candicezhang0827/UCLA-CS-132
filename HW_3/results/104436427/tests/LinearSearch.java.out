--- Error (ProgramFault) ---
call to "LS.Print": incorrect number of arguments in function call (expecting: 1, found: 2)
  Main() line 14
  LS.Start(<Heap.0>+0 10) line 24
